src/main/java/com/redhat/devtools/lsp4ij/features/semanticTokens/viewProvider/LSPSemanticTokenElementType.java

```diff
@@ -0,0 +1,27 @@
+package com.redhat.devtools.lsp4ij.features.semanticTokens.viewProvider;
+
+import com.intellij.psi.tree.IElementType;
+import com.redhat.devtools.lsp4ij.LSPIJEditorUtils;
+import org.jetbrains.annotations.NonNls;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * LSP semantic token element types.
+ */
+class LSPSemanticTokenElementType extends IElementType {
+    // These are the concrete element types that we can derive from semantic token types and modifiers
+    static final LSPSemanticTokenElementType DECLARATION = new LSPSemanticTokenElementType("DECLARATION");
```

Comment:
Now that we have actual `PsiElement`s, we need `IElementType`s for them. These are the known element types for an LSP4IJ TextMate file backed by semantic tokens.